pytket/pytket/qasm/qasm.py

```diff
@@ -335,18 +335,84 @@ def mixedcall(self, tree: List) -> Iterator[CommandDict]:
         except StopIteration:
             args = next_tree
             pars = []
-        params = [f"({par})/pi" for par in pars]
+
+        params = [
+            f"({par})/pi" for par in pars
+        ]  # do we want to do this for all parameters?
+
+        treat_as_barrier = [
```

Comment:
I am not happy with this solution to have the list of the gates in here which we want to have converted to a barrier at the end. The problem is that the gates are all marked as opaque like others in the dev header file which we want to have handled differently, so I have not found a solution which is not having a hard coded list in here. I think the better idea would be to have pytket special gates for sleep, group and order like for the other opaque gates, but that would have been a bigger changes which we maybe should discuss in a bigger group?

Even with not being happy with this, I would suggest adding the for now as suggested in this PR.